README.rst

```diff
@@ -2,12 +2,15 @@
 Kayobe Configuration for "A Universe from Nothing: Containerised OpenStack deployment using Kolla, Ansible and Kayobe"
 ======================================================================================================================
 
-This repository provides configuration for the `Kayobe
-<https://kayobe.readthedocs.io/en/latest>`__ project. It is based on the
+Originally created for the Denver 2019 Openstack Infrastructure Summit as a
```

Comment:
It was actually created for a Scientific OpenStack Hackathon earlier this year, but that's a minor point.  I don't think we need specify.  Consider instead something along the lines of *This guide was originally created as a workshop to configure, deploy and get hands-on with Kayobe...*

Reply:
Amended the README with the suggested changes.